fbw-a32nx/src/systems/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -217,7 +217,7 @@ export class EfisSymbols<T extends number> {
             // eslint-disable-next-line no-loop-func
             const upsertSymbol = (symbol: NdSymbol): void => {
                 if (DEBUG) {
-                    console.time(`upsert symbol ${symbol.databaseId}`);
+                    // console.time(`upsert symbol ${symbol.databaseId}`);
```

Comment:
I would either uncomment this, or remove the piece of code entirely if it's not needed anymore.

Reply:
done